src/gui/dbsettings/DatabaseSettingsWidgetGeneral.cpp

```diff
@@ -57,7 +57,7 @@ void DatabaseSettingsWidgetGeneral::initialize()
         m_ui->historyMaxSizeSpinBox->setValue(historyMaxSizeMiB);
         m_ui->historyMaxSizeCheckBox->setChecked(true);
     } else {
-        m_ui->historyMaxSizeSpinBox->setValue(Metadata::DefaultHistoryMaxSize);
+        m_ui->historyMaxSizeSpinBox->setValue(qRound(Metadata::DefaultHistoryMaxSize / qreal(1048576)));
```

Comment:
For better readability I would define as `1024 * 1024`. It will not impact on runtime performance because the compiler will multiply it on build time.

Reply:
Didn't know compilers can do that, assumed it would be best to keep the result as i saw it already written that way on line 56.
I will fix this shortly